Makefile

```diff
@@ -70,15 +70,9 @@ endif
 	  exit 1; \
 	}
 
-	type govendor >/dev/null 2>&1|| { \
-	  echo "\033[1;33mgovendor is not installed. See https://github.com/kardianos/govendor\033[m"; \
-	  echo "$$ go get -u github.com/kardianos/govendor"; \
-	  exit 1; \
-	}
-
 	type gox >/dev/null 2>&1 || { \
 	  echo "\033[1;33mGox is not installed. See https://github.com/mitchellh/gox\033[m"; \
-	  echo "$$ go get github.com/mitchellh/gox"; \
+	  echo "$$ go install github.com/mitchellh/gox@latest"; \
```

Comment:
https://golang.org/doc/go-get-install-deprecation

Reply:
I wonder if it would be safer to pin this to a specific version? @tillberg what do you think?

Reply:
Is this command run in CI? If so, then this should use a cryptographic hash for the version reference. Otherwise, I think I'm hesitant to weigh in one way or the other :).

Reply:
It's not part of CI, but it is part of the manual build/release process. I like the idea of making the version explicit. Added `tools.go` in cb74f9f based on [this technique](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module), which seems to be as good as it gets while we wait for golang/go#48429.

This could've been messy if `gox` shared any dependencies with `remote_syslog2` itself, but I confirmed there weren't any shared dependencies.